### PR TITLE
Created basic chess board

### PR DESCRIPTION
**Pull request by hughesbm**
- Board/squares/pieces resize responsively
- Colors/sizes/ratios/etc are easily modified with sass variables
- Board squares have data elements with x and y coordinates
- There is a board-rotate class that can be applied at some point in the future; this would allow both players to perceive the board with their side at the bottom, without complicating how the coordinate data elements are assigned to squares
- I thought about adding textures to the squares but I think I prefer flat colors. Any thoughts?

For now, I included the board on the index page so it's easier to preview. It'll be easy enough to remove later.

### CODE REVIEW

**Comment by hughesbm** [comment]
Oh, right. I left some piece images on the board as a demonstration. Obviously those won't be left there permanently.

File: app/views/games/_game_board.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="board">
+  <% 8.times do |y| %>
```
Comment (mscoutermarsh) [inline app/views/games/_game_board.html.erb]: Nice - I like how you did this. Will be easy to add to later.

**Comment by mscoutermarsh** [comment]
Code looks :+1:.

Can you add a screenshot?

Before you merge it, will need to get it up to date with master. To do that, I recommend `rebasing`.

```
git checkout master
git pull
# now master is up to date
git checkout board
git rebase master

# since rebase changes git history, will have to force push your branch
git push origin board -f
```

After that is done, wait until travisci says the branch is green, then can merge it. :)

**Comment by hughesbm** [comment]
Screenshots:

With pieces:
w/ pieces

Without pieces:
w/o pieces

**Pull request merged**